### ISSUE TITLE
Autoprefixer for gulp style task

### ISSUE THREAD

**Issue by Schnueggel** [doroppu/yoso-gulp#1]
Can we add autoprefixing for styles.

For example:

```javascript
.pipe( autoprefixer({
            browsers: ['last 2 versions'],
            cascade: false
        }) )
```

**Comment by lukas-zech-software** [comment]
The vendor prefixes are added on __build-release-styles__ but should also be added on __build-styles__

**Comment by lukas-zech-software** [comment]
Fixed in 1.0.1
